mod-orders-storage/schemas/acquisitions_unit_membership.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Relationship between a user and acquistionUnit",
```

Comment:
nitpick. I think `acquistionUnit` should be splitted into separate words

Reply:
done